modules/hmtnote/main.nf

```diff
@@ -0,0 +1,65 @@
+process HMTNOTE {
+    tag "$meta.id"
+    label 'process_low'
+
+    // TODO nf-core: List required Conda package(s).
+    //               Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    //               For Conda, the build (i.e. "h9402c20_2") must be EXCLUDED to support installation on different operating systems.
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
```

Comment:
Please remove all comments and TODO's

Reply:
Thank you for reviewing, I have done the changes as mentioned